relbench/tasks/event.py

```diff
@@ -42,9 +31,15 @@ def make_table(self, db: Database, timestamps: "pd.Series[pd.Timestamp]") -> Tab
         df = duckdb.sql(
             f"""
             SELECT
+                event_info.timestamp,
+                event_info.user,
+                event_info.target
+            FROM
+            (SELECT
                 t.timestamp,
                 event_attendees.user_id AS user,
-                SUM(CASE WHEN event_attendees.status IN ('yes', 'maybe') THEN 1 ELSE 0 END) AS target
+                SUM(CASE WHEN event_attendees.status IN ('yes', 'maybe') THEN 1 ELSE 0 END) AS target,
+                LAG(SUM(CASE WHEN event_attendees.status IN ('yes', 'maybe') THEN 1 ELSE 0 END), 1) OVER (PARTITION BY event_attendees.user_id ORDER BY t.timestamp) AS prev_target
```

Comment:
what does this do?

Reply:
Creates the `prev_target` column, which represents if a user has previously attended an event in the last 7 days or not.

Reply:
Interesting. It's somehow pretty hard to read that off from this SQL query :)